uxarray/plot/accessor.py

```diff
@@ -77,11 +87,32 @@ def points(self, element="nodes", backend=None, **kwargs):
         else:
             raise ValueError(f"Unsupported element {element}")
 
+        # Set default kwargs for better visualization
+        if "color" not in kwargs:
+            kwargs["color"] = "blue"
+        if "size" not in kwargs:
+            kwargs["size"] = 50
+        if "title" not in kwargs:
+            if self._uxgrid.n_face == 0 and self._uxgrid.n_edge == 0:
+                kwargs["title"] = "Points"
+            else:
+                kwargs["title"] = f"{element.capitalize()}"
+
+        # Set projection defaults
+        if "projection" not in kwargs:
+            kwargs["projection"] = ccrs.PlateCarree()
+        if "crs" not in kwargs:
+            if "projection" in kwargs:
+                central_longitude = kwargs["projection"].proj4_params.get("lon_0", 0.0)
+            else:
+                central_longitude = 0.0
+            kwargs["crs"] = ccrs.PlateCarree(central_longitude=central_longitude)
+
         verts = {"lon": lon, "lat": lat}
 
         points_df = pd.DataFrame.from_dict(verts)
 
-        return points_df.hvplot.points("lon", "lat", **kwargs)
+        return points_df.hvplot.points("lon", "lat", geo=True, **kwargs)
```

Comment:
I've seen cases where we needed to use `geo=True`, but what is the expectation with defaulting to it for the function? Also what is the extent, I mean I'd be curious if it'd impact how the plots behave overall?

Reply:
Have you encountered specific use cases where geo=False would be preferred for lon/lat data?

Reply:
I see often that the users' workflows include plots of the data without any projection to quick inspect it in addition to many they generate with projections. 

I think I really like the boiler-plate of `polygons()` in this same file, and you may want to use the same aprroach here and in edges:

```
if projection is not None:
    kwargs["projection"] = projection
    kwargs["geo"] = True
```

I think this really addresses your point well: If projection is desired, then make `geo=True`. The only caveat here would be that if the user also specified `geo=False` in `kwargs`, we are forcing it to `True`, but for a good reason, and that shouldn't be an issue I think. 

Thoughts?

Reply:
> 
> Thoughts?

Agreed. I'll pushes changes soon. Testing all notebook plots, I broke some, fixing them now.

Reply:
done